var/spack/repos/builtin/packages/json-c/package.py

```diff
@@ -19,8 +19,9 @@ class JsonC(AutotoolsPackage):
 
     parallel = False
 
-    @when('@0.12.1 %gcc@7:')
+    @when('@0.12:0.12.99 %gcc@7:')
```

Comment:
Not sure if this change is necessary, as there is only a single release in your range.

Reply:
I agree.